ui/pages/Project/components/RnaSeqOutliers.jsx

```diff
@@ -53,11 +53,13 @@ class RnaSeqOutliersGraph extends React.PureComponent {
       .append('g')
 
     dataPoints.append('circle')
-      .attr('cx', d => x(d[xField]))
-      .attr('cy', d => y(d[yField]))
-      .attr('r', 3)
-      .style('fill', 'None')
+      .attr('cx', d => x(d.zScore))
+      .attr('cy', d => y(d.pValue))
+      .attr('r', d => (d.deltaPsi === undefined ? 3 : r(Math.abs(d.deltaPsi))))
+      .style('fill-opacity', '0%')
       .style('stroke', d => (d.isSignificant ? 'red' : 'lightgrey'))
+      .append('svg:title')
+      .text(d => `${genesById[d.geneId] ? genesById[d.geneId].geneSymbol : d.geneId}\nZ = ${d.zScore.toPrecision(2)}\n${'P-value = 10^-'}${(-Math.log10(d.pValue)).toPrecision(2)}`)
```

Comment:
this will need to use the `xField` and `yField` logic instead of hardcoding fields